workers_launch_template_mixed.tf

```diff
@@ -183,7 +183,7 @@ resource "aws_autoscaling_group" "workers_launch_template_mixed" {
           var.worker_groups_launch_template_mixed[count.index],
           "autoscaling_enabled",
           local.workers_group_defaults["autoscaling_enabled"],
-        ) == 1 ? "enabled" : "disabled"}"
+        ) == "true" ? "enabled" : "disabled"}"
```

Comment:
There is no need to do`== "true"`, in expression `var ? this : that`, `var` must be either bool or string that can be converted to bool, e.g. `"true"` or `"false"`.